dev-util/clion/clion-2019.2.1.ebuild

```diff
@@ -9,11 +9,14 @@ DESCRIPTION="A complete toolset for C and C++ development"
 HOMEPAGE="https://www.jetbrains.com/clion"
 SRC_URI="https://download.jetbrains.com/cpp/CLion-${PV}.tar.gz -> ${P}.tar.gz"
 
-LICENSE="IDEA
-	|| ( IDEA_Academic IDEA_Classroom IDEA_OpenSource IDEA_Personal )"
+LICENSE="Apache-1.1 Apache-2.0 BSD BSD-2 CC0-1.0 CDDL-1.1 CPL-0.5 CPL-1.0
+	EPL-1.0 EPL-2.0 GPL-2 GPL-2-with-classpath-exception GPL-3 ISC
+	JDOM LGPL-2.1+ LGPL-3 MIT MPL-1.0 MPL-1.1 OFL oromatcher
+	public-domain PSF-2 sun-bcla-javahelp UoI-NCSA ZLIB
```

Comment:
What files in the package are licensed under sun-bcla-javahelp?
Javahelp used to be licensed under GPL-2-with-classpath-exception, and was recently relicensed to CDDL-1.1.

Reply:
you're right, my fault.

Reply:
oromatcher is gone as well? Was that for lib/oro-2.0.8.jar (which really is Apache-1.1)?